### PR TITLE
Refactor/make-optional-env-vars-dynamic

### PR DESCRIPTION
**Pull request by Ahmedhossamdev**
fixes: #117

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,7 +8,7 @@ This is the next-generation OneBusAway web application, built on top of [SvelteK
 npm install
 cp .env.example .env
 # edit .env with your editor of choice
-npm run dev
+npm run devPUBLIC_NAV_BAR_LINKS
```
Comment (aaronbrethorst) [inline README.md:11]: typo?

**Review by aaronbrethorst — CHANGES_REQUESTED** [review]
looks like there's a small change to make in the README, but otherwise this is ready to go. Thanks for jumping on this!

File: README.md (hunk 1 above)
Comment (Ahmedhossamdev) [reply README.md]: My bad

**Review by aaronbrethorst — APPROVED** [review]
👍

**Pull request merged**